ros_buildfarm/templates/snippet/credential_bindings.xml.em

```diff
@@ -0,0 +1,20 @@
+    <org.jenkinsci.plugins.credentialsbinding.impl.SecretBuildWrapper plugin="credentials-binding@@1.20">
```

Comment:
This is really about the snippet filename rather than the first line but should this be `credentials_binding.xml.em` to match the plugin name `credentials-binding`?

Reply:
I mostly just wanted to distance the name from the deprecated snippet, `credentials_binding_plugin`. I'm fine with `credentials_binding` as well.